engine/language-server/src/test/scala/org/enso/languageserver/libraries/ComponentGroupsResolverSpec.scala

```diff
@@ -280,7 +280,8 @@ object ComponentGroupsResolverSpec {
       maintainers          = Nil,
       edition              = None,
       preferLocalLibraries = true,
-      componentGroups      = Some(componentGroups)
+      componentGroups      = Some(componentGroups),
+      jvm                  = None
```

Comment:
Is it important to have a tri-state value for `jvm`? Isn't binary `Boolean` enough?

Reply:
It's not enough if we want to be backwards compatible with configs that don't have that field.